heron/scheduler-core/src/java/com/twitter/heron/scheduler/utils/SchedulerConfigUtils.java

```diff
@@ -78,7 +78,7 @@ public static Config loadConfig(
       Boolean verbose,
       TopologyAPI.Topology topology) {
 
-    return Config.expand(
+    return Config.toRemoteMode(
```

Comment:
Yeah, I don't love the terminology here of local, remote or sandbox and would like to change it. It's really about how Heron is installed. It's either installed on the system, or bundled in a distribution. I was thinking about "installed" and "bundled" modes instead, but those aren't great either. Suggestions welcome.